tests/ci/cdk/cdk/aws_lc_ec2_test_framework_ci_stack.py

```diff
@@ -84,10 +84,14 @@ def __init__(self,
                                 iam.ManagedPolicy.from_aws_managed_policy_name("AmazonSSMManagedInstanceCore"),
                                 iam.ManagedPolicy.from_aws_managed_policy_name("CloudWatchAgentServerPolicy")
                             ])
-        iam.CfnInstanceProfile(scope=self, id="{}-ec2-profile".format(id),
+        cfn_instance_prof = iam.CfnInstanceProfile(scope=self, id="{}-ec2-profile".format(id),
                                roles=["{}-ec2-role".format(id)],
                                instance_profile_name="{}-ec2-profile".format(id))
 
+        cfn_ec2_role = iam.CfnRole( self, "CfnEc2Role",
```

Comment:
How does this work? Is this a role with no permissions, and it allows EC2 to assume it?

Reply:
Fixed in latest commit